js/src/edit-block/helpers/convertToString.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Internal dependencies
+ */
+import { TEXT_ARRAY_DELIMITER } from '../constants';
+
+/**
+ * Converts the setting to a string.
+ *
+ * @param {Array} settingValues The settings values.
+ * @return {string} The setting converted to a string.
+ */
+const convertToString = ( settingValues ) => {
```

Comment:
Small thing, but it might be nice to be more explicit with the name here, like `convertSettingsArraytoString` or similar since that's what it's made specifically to do. Maybe something similar for the names of the other helpers?

Reply:
Good point. It's not to convert anything to a string, just a settings `array`.

Reply:
Applied in https://github.com/studiopress/genesis-custom-blocks/pull/41/commits/cc218836085b81b4d5a91fc720ca86e6536a6b66